es/outreach.tex

```diff
@@ -373,27 +377,33 @@ \chapter{Difusión}\label{s:outreach}
 
 \seclbl{Cambio académico}{s:outreach-schools}
 
-Todo el mundo tiene miedo a lo desconocido y a pasar vergüenza.En consecuencia,
+Todo el mundo tiene miedo a lo desconocido y a pasar vergüenza. En consecuencia,
 la mayoría de la gente prefiere fracasar que cambiar.
 Por ejemplo,
 Lauren Herckis investigó \index{Herckis, Lauren}
-\hreffoot{https://www.insidehighered.com/news/2017/07/06/anthropologist-studies-why-professors-dont-adopt-innovative-teaching-methods}{por qué los/las profesores/as de las universidades no adoptan mejores metodos de enseñanza}.
+\hreffoot{https://www.insidehighered.com/news/2017/07/06/anthropologist-studies-why-professors-dont-adopt-innovative-teaching-methods}
+{por qué los/las docentes de nivel universitario no adoptan mejores metodos de enseñanza}.
 Lauren halló que el principal motivo es el miedo a parecer estúpido/a delante de  los estudiantes; 
-Las razones secundarias fueron
-preocupación de que los inevitables impactos al cambiar los métodos de enseñanza pudieran afectar las evaluaciones de los cursos (que  a su vez podría afectar promociones y cargos) 
-y el deseo de la gente de seguir emulando a los/las docentes que los inspiraron. 
+Las otras razones fueron:
+preocupación que el cambio de los métodos de enseñanza pudieran afectar las evaluaciones de los cursos de forma negativa
+(que  a su vez podría afectar promociones y cargos) 
+y el deseo de la gente de seguir emulando a los/las docentes que les inspiraron. 
 
 No tiene sentido discutir si estas cuestiones son ``reales'' o no:
-los/las profesores creen que son reales,
+los/las docentes creen que son reales,
 por lo que cualquier plan para trabajar con el personal docente de las universidades necesita tenerlas en cuenta\footnote{
-Así como la prevalencia de la mentalidad fija entre los/las profesores cuando se trata de, la creencia de que algunas personas son ``solo mejores profesores'' }.
+Así como la prevalencia de la mentalidad fija entre los/las docentes cuando se trata de, 
```

Comment:
Me resulta poco claro, falta una parte me parece y hay algo con el concepto "fixed mindset". Copio el inglés: _And the prevalence of fixed mindset among faculty when it comes to teaching, i.e. the belief that some people are “just better teachers.”_
Puede ser que  más que "mentalidad fija" sea "mentalidad cerrada" para nosotrxs? Ejemplo: https://www.brainpickings.org/2014/01/29/carol-dweck-mindset/  

```suggestion
Así como la prevalencia de una mentalidad cerrada entre los/las docentes cuando se trata de enseñanza, es decir, 
```

Reply:
Yani, esta sugerencia no te gustó por algo o sólo se pasó? Estoy revisando el capítulo y me vuelve a saltar como que hay algo que falta al final de la oración.

Reply:
Se me pasó Nati y si, parece que le falta algo, ahi miro en el texto completo

Reply:
tranqui! yo lo agrego!!